settings.yml.sample

```diff
@@ -1,10 +0,0 @@
-host:
```

Comment:
Should this have been a `git mv` rather than a normal `mv`? Then we could save the commit history for the file.

Reply:
StackOverflow [suggests](https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history) that `git mv` wouldn't do this, and my empirical attempt to compare the two suggests the same. If you know the magic incantation here I'm happy to try it though.